packages/dosespot-react/src/index.test.ts

```diff
@@ -0,0 +1,7 @@
+import { useDoseSpotNotifications } from '.';
+
+describe('Index', () => {
+  test('Expected exports', () => {
+    expect(useDoseSpotNotifications).toBeDefined();
```

Comment:
Should we test `useDoseSpotIframe` as well?

Reply:
Yeah, I'm writing tests now.

I was copying ML's approach with `@medplum/health-gorilla-react`, where he just verifies that at least one export is available.  My assumption was that the intent was just to verify the barrel is working as expected.  In this case, where we only have 2 exports, maybe it's worth it to be exhaustive.

Reply:
✅